### PR TITLE
Add screen to view instance logs

### PR DESCRIPTION
**Pull request by PierreBesson**
This is a collaborative PR with @pascalgrimaud !

screenshot at 2017-07-24 12-37-23

### CODE REVIEW

**Comment by jdubois** [comment]
Looks great !
But the logs/loggers menus aren't very clear, we should find better names.

**Comment by PierreBesson** [comment]
@jdubois To me this is the best I can name them. Much better than the previous, "logfile" for viewing logs and "logs" for viewing loggers.

**Comment by pascalgrimaud** [comment]
I'd rather prefer `Bottom` and `Top` as external button than inside the log text

I'm fine with Logs and Loggers name. But to be more clear, I propose:
- View Logs
- Loggers (Loggers Configuration would be too long)

**Comment by pascalgrimaud** [comment]
Edit: it's better inside the log text in fact, but at the right of the log (same as Travis)
Need an UX designer here  (@deepu105 ?) :stuck_out_tongue:

**Comment by PierreBesson** [comment]
screenshot at 2017-07-25 11-08-14

**Comment by pascalgrimaud** [comment]
LGTM

File: src/main/webapp/app/admin/logfile/logfile.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div>
+    <h2>View logs</h2>
+
+    <jhi-route-selector></jhi-route-selector>
+
+    <div class="to-bottom" *ngIf="logtxt" (click)="scrollToBottom()">&#8675;Bottom</div>
```
Comment (deepu105) [inline src/main/webapp/app/admin/logfile/logfile.component.html]: why dont you use some nice icon from font awesome here? same for to-top as well `arrow-circle-up` and `arrow-circle-down` would be good choice

Comment (PierreBesson) [inline src/main/webapp/app/admin/logfile/logfile.component.html]: Good idea. I've just done it.
screenshot from 2017-08-02 16-17-04

**Comment by pascalgrimaud** [comment]
Good job @PierreBesson !

**Pull request merged**